### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by 5andi5**
This PR adds functionality for reading person's date of birth from a certificate.
Also fixes some issues and inconveniences.

### CODE REVIEW

**Comment by 5andi5** [comment]
@LeKristapino Could You review these changes, please?

**Comment by LeKristapino** [comment]
@5andi5 Sorry, will review & merge

File: lib/smart_id/api/request.rb [hunk 1]
```diff
@@ -5,9 +5,9 @@ class Request
     DEMO_BASE_URL = "https://sid.demo.sk.ee/smart-id-rp/v1/"
     PRODUCTION_BASE_URL = "https://rp-api.smart-id.com/v1/"
 
-    DEMO_SSL_KEY = "QLZIaH7Qx9Rjq3gyznQuNsvwMQb7maC5L4SLu/z5qNU="
-    PROD_KEY_EXPIRY = Date.new(2020,11,5)
-    PRODUCTION_SSL_KEY = "l2uvq6ftLN4LZ+8Un+71J2vH1BT9wTbtrE5+Fj3Vc5g="
+    # DEMO_SSL_KEY = "QLZIaH7Qx9Rjq3gyznQuNsvwMQb7maC5L4SLu/z5qNU="
```
Comment (LeKristapino) [inline lib/smart_id/api/request.rb]: I guess if this is not needed we can delete

File: lib/smart_id/api/request.rb [hunk 2]
```diff
@@ -40,49 +40,49 @@ def self.execute(method:, uri:, params:)
       end
     end
 
-    def maybe_warn_of_ssl_key_expiry
-      if (PROD_KEY_EXPIRY - Date.today).to_i < 60
-        @logger.warn("[Smart-id-Ruby] SSL KEY for security checks will soon expire, please update to newer version of this gem")
-      end
-    end
+    # def maybe_warn_of_ssl_key_expiry
```
Comment (LeKristapino) [inline lib/smart_id/api/request.rb]: Same here

**Comment by 5andi5** [comment]
@LeKristapino Thanks a lot

**Review by LeKristapino — CHANGES_REQUESTED** [review]
@5andi5 Please remove the commented out code if it is not needed anymore.

**Comment by 5andi5** [comment]
> @5andi5 Please remove the commented out code if it is not needed anymore.

@LeKristapino Issue fixed

**Pull request merged**